NitroxClient/MonoBehaviours/Debugging/SceneDebugger.cs

```diff
@@ -0,0 +1,253 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Globalization;
+using System.Linq;
+using System.Text;
+using NitroxClient.Unity.Helper;
+using UnityEngine;
+using UnityEngine.SceneManagement;
+
+namespace NitroxClient.MonoBehaviours.Debugging
+{
+    public class SceneDebugger : BaseDebugger
+    {
+        private Scene selectedScene;
+        private GameObject selectedObject;
+        private Vector2 hierarchyScrollPos;
+        private Vector2 gameObjectScrollPos;
+        private Color paleGreen = new Color(152, 251, 152);
+        private const int MAX_HIERARCHY_ENTRIES = 100;
+
+        public void Awake()
+        {
+            Hotkey = KeyCode.S;
+            HotkeyControlRequired = true;
+            WindowRect.width = 400;
+            SkinCreationOptions = GUISkinCreationOptions.DERIVEDCOPY;
+
+            Tabs.AddRange(new[]
+            {
+                "Scenes",
+                "Hierarchy",
+                "GameObject"
+            });
+        }
+
+        public override void DoWindow(int windowId)
+        {
+            switch (ActiveTab)
```

Comment:
Can `ActiveTab` be changed to an `enum` type to better distinguish the possible pages?

Reply:
The initial plan was as you say but because I wanted to support multiple debuggers/windows you would need to define an enum per debugger. Still it's better than having to remember numbers.

Reply:
Exactly, I think the extra burden of defining them somewhere outweighs numbers that are easy to forget (and may cause confusion). Enums are basically numbers with a name anyway.